Branch-TestBed-Swift/TestBed-Swift/AppDelegate.swift

```diff
@@ -98,14 +101,20 @@ class AppDelegate: UIResponder, UIApplicationDelegate, AdjustDelegate, AppsFlyer
                     if clickedBranchLink {
                         let nc = self.window!.rootViewController as! UINavigationController
                         let storyboard = UIStoryboard(name: "Content", bundle: nil)
-                        let contentViewController = storyboard.instantiateViewController(withIdentifier: "Content") as! ContentViewController
+                        let contentViewController = storyboard.instantiateViewController(
+                            withIdentifier: "Content"
+                        ) as! ContentViewController
                         nc.pushViewController(contentViewController, animated: true)
                         
-                        let referringLink = paramsDictionary["~referring_link"] as! String
```

Comment:
Was this line causing the crash? Is there chance we get  anything other than type String for `referring link`?

Reply:
It was mis-cast.